less/button.less

```diff
@@ -88,94 +90,80 @@ span.menuButton {
   &.primaryButton,
   &.actionButton,
   &.wideButton,
-  &.secondaryButton,
-  &.secondaryAltButton,
   &.subtleButton,
   &.whiteButton {
-    .buttonCommon;
+    color: #fff;
+    line-height: 1.25;
+    padding-top: 5px;
+    padding-bottom: 5px;
     width: auto;
     height: auto;
   }
 
+  &.primaryButton,
+  &.actionButton,
+  &.subtleButton,
+  &.whiteButton {
+    padding-right: 16px;
+    padding-left: 16px;
+  }
+
   &.primaryButton {
     background: linear-gradient(@braveLightOrange, @braveOrange);
-    border: 2px solid transparent;
-    border-top: 2px solid @braveLightOrange;
-    border-bottom: 2px solid @braveOrange;
+    border: 1px solid transparent;
+    border-top: 1px solid @braveLightOrange;
+    border-bottom: 1px solid @braveOrange;
     box-shadow: @buttonShadow;
     font-weight: 400;
 
     &:hover {
-      border: 2px solid white;
+      border: 1px solid white;
       color: white;
       cursor: pointer;
     }
   }
 
-  &.actionButton,
-  &.wideButton {
+  &.actionButton {
     background-color: #4099FF;
   }
 
   &.wideButton {
-    display: block;
-    margin-right: 0px;
-    margin-left: 0px;
     width: auto;
-    padding: 3px 35px;
+    padding-right: 35px;
+    padding-left: 35px;
   }
 
   &.whiteButton {
     background: linear-gradient(white, @lightGray);
-    border: 2px solid white;
+    border: 1px solid white;
     box-shadow: @buttonShadow;
     cursor: pointer;
-    font-weight: normal;
-    font-style: normal;
     color: @darkGray;
+
+    &:hover {
+      border: 1px solid @gray;
+    }
   }
 
   &.inlineButton {
     display: inline;
   }
 
-  &.secondaryButton {
-    background-color: #eee;
-  }
-
-  &.secondaryAltButton {
-    background-color: #fff;
-    color: @braveOrange;
-    border: 1px solid #ccc;
-  }
-
   &.subtleButton {
     background-color: #ccc;
     font-size: 14px;
   }
+}
 
-  &.tactileButton {
-    display: block;
-    min-width: 80px;
-    height: 2em;
-    line-height: 2em;
-    margin: 0.5em 0;
-    vertical-align: middle;
-    text-align: center;
-    border: solid 1px @braveMediumOrange;
-    background: @braveMediumOrange;
-    border-radius: @borderRadius;
-    color: white;
-    box-shadow: @lightBoxShadow;
-    padding-left: 1em;
+// for about:preferences
+.prefBody {
+  .settingsList > .settingItem + button {
```

Comment:
Because the button should be placed under `<SettingCheckbox>`. For example: https://github.com/brave/browser-laptop/pull/5916/files#diff-e3eeb751016b2ce9f8278efce585a461R1519